docs/markdown/snippets/subproject_and_override_program_native.md

```diff
@@ -0,0 +1,13 @@
+## subproject() and find_program() now support the native keyword argument
```

Comment:
Doesn't `find_program()` already support the `native` keyword?

Reply:
Yes, oops. That should be meson.override_find_program